plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/ProjectFileParsingManager.kt

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.codewhisperer.codescan
```

Comment:
this is now just a copy of `FeatureDevSessionContext`

CodeScan and FeatureDev should be sharing this logic

Reply:
I thought i'm supposed to dedupe this because of the import issue?